### DISCUSSION TITLE
Exposing ReactPHP Server port

### DISCUSSION THREAD

**Discussion by Modicrumb** [reactphp/reactphp#485] [Q&A] [answered]
I want to run a simple ReactPHP TCP server on port 8082. I am not sure how to expose this port to computers outside the server. I'm using AWS and have allowed traffic from 8082 from anywhere to reach the server. I do want to also tie it to a domain if that makes sense. I want to be able to curl domain.com:8082 from my local terminal and have it hit my ReactPHP server. I do have apache running on the server and am not sure what to do to be able to achieve this.

**Comment by Modicrumb** [comment, marked as answer]
Got this sorted, didn't need to use apache, just needed to set the application code on the server to listen on the internal private ipv4 address of the server. I can now access it from outside.

**Reply by SimonFrings** [reply]
Hey @Modicrumb , great that you found a solution for this :+1:

As a side note: You want to use Apache in cases where you're speaking HTTP, in this case you're only using TCP from what you described above. As you already figured out, you don't need Apache in this case :)
